### ISSUE TITLE
`redundantFileprivate` rule breaks build 

### ISSUE THREAD

**Issue by calda** [nicklockwood/SwiftFormat#1293]
The `redundantFileprivate` rule is incorrectly converting this `fileprivate` declaration to `private`, breaking the build:

### Before

```swift
extension [String] {
  fileprivate func fileprivateMember() { }
}

extension Namespace {
  func testCanAccessFileprivateMember() {
    ["string", "array"].privateMember()
  }
}
```

### After running `swiftformat test.swift --rules redundantFileprivate --swiftversion 5`

```swift
extension [String] {
  private func fileprivateMember() { }
}

extension Namespace {
  func testCanAccessFileprivateMember() {
    // error: 'fileprivateMember' is inaccessible due to 'private' protection level
    ["string", "array"].fileprivateMember()
  }
}

**Comment by nicklockwood** [comment]
I've fixed this specific issue, but there are still some related cases I need to handle

**Comment by nicklockwood** [comment]
@calda fixed in 0.50.3
